### PR TITLE
Add admin user index endpoint and system admin UI

### PR DESCRIPTION
**Pull request by garrettqmartin8**
## Description

API endpoint and UI for viewing admin users in the system admin app.

## Setup

- `make server_run`
- `make client_run`
- Log into the admin app and click on "Admin Users" in the sidebar to see a list of the admin users in the system

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/167854820) for this change

### CODE REVIEW

File: pkg/handlers/adminapi/api.go [hunk 1]
```diff
@@ -81,5 +82,13 @@ func NewAdminAPIHandler(context handlers.HandlerContext) http.Handler {
 		query.NewQueryFilter,
 		pagination.NewPagination,
 	}
+
+	adminAPI.AdminUsersIndexAdminUsersHandler = IndexAdminUsersHandler{
```
Comment (lynzt) [inline pkg/handlers/adminapi/api.go]: `OfficeIndexOfficeUsersHandler` is seen above... 
`AdminUsersIndexAdminUsersHandler` - keeping inline w/ this naming pattern would make this: `AdminIndexAdminUsersHandler`?

File: pkg/handlers/adminapi/api.go [hunk 2]
```diff
@@ -32,26 +33,26 @@ func NewAdminAPIHandler(context handlers.HandlerContext) http.Handler {
 
 	adminAPI.OfficeIndexOfficeUsersHandler = IndexOfficeUsersHandler{
```
Comment (lynzt) [inline pkg/handlers/adminapi/api.go]: can you tell me the naming convention of these handlers?
for `OfficeIndexOfficeUsersHandler`, could you ever have a: `OfficeIndexAdminUsersHandler` or `AdminIndexOfficeUsersHandle`? 
if not, the duplicated `Office` seems redundant?

File: pkg/handlers/adminapi/admin_users.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+// IndexAdminUsersHandler returns a list of office users via GET /admin_users
+type IndexAdminUsersHandler struct {
+	handlers.HandlerContext
+	services.AdminUserListFetcher
+	services.NewQueryFilter
+	services.NewPagination
+}
+
+// Handle retrieves a list of office users
+func (h IndexAdminUsersHandler) Handle(params adminuserop.IndexAdminUsersParams) middleware.Responder {
```
Comment (lynzt) [inline pkg/handlers/adminapi/admin_users.go]: I think I'm getting confused between `officeUsers` and `adminUsers` in this handler... 
should `officeUsers` be `adminUsers`...
if not - I might need a quick run down of how they relate here...

File: pkg/services/mocks/AccessCodeClaimer.go [hunk 4]
```diff
@@ -2,11 +2,14 @@
...
+import (
+	mock "github.com/stretchr/testify/mock"
+	models "github.com/transcom/mymove/pkg/models"
 
-import uuid "github.com/gofrs/uuid"
-import validate "github.com/gobuffalo/validate"
+	uuid "github.com/gofrs/uuid"
+
+	validate "github.com/gobuffalo/validate"
+)
```
Comment (lynzt) [inline pkg/services/mocks/AccessCodeClaimer.go]: is this better for performance or just bringing it inline w/ what we have for import stmts?

File: pkg/handlers/adminapi/api.go (hunk 1 above)
Comment (garrettqmartin8) [reply pkg/handlers/adminapi/api.go]: This comes from how our Swagger configuration affects the generated code. In particular, the `tag` field. Honestly, I think `AdminUsers` is more descriptive than `Admin`, so I may change the `office` tag to `office_users`.

File: pkg/handlers/adminapi/api.go (hunk 2 above)
Comment (garrettqmartin8) [reply pkg/handlers/adminapi/api.go]: The verbose names come from the code generation based on our Swagger config. I think it's a combination of the `tag` field and the `operationID`. It does make for a pretty verbose setup, but I'm not sure about making the actual Swagger config less descriptive.

File: pkg/handlers/adminapi/admin_users.go (hunk 3 above)
Comment (garrettqmartin8) [reply pkg/handlers/adminapi/admin_users.go]: That's just me not properly converting variable names :)

File: pkg/services/mocks/AccessCodeClaimer.go (hunk 4 above)
Comment (garrettqmartin8) [reply pkg/services/mocks/AccessCodeClaimer.go]: This was generated - I believe it's just a shorthand so you don't have to write `import` over and over again. I think `gofmt` or `goimports` does this automatically.

**Pull request merged**